docs/user/dql/functions.rst

```diff
@@ -3137,17 +3137,17 @@ Description
 
 ``wildcard_query(field_expression, query_expression[, option=<option_value>]*)``
 
-The wildcard_query function maps to the wildcard_query query used in search engine. It returns documents that match provided text in the specified field.
-Supported wildcard characters can be found here: https://opensearch.org/docs/latest/opensearch/query-dsl/term/#wildcards
-SQL wildcard % is converted to * as well as _ to ?. You may include % and _ in the search by escaping with a backslash prefix.
+The `wildcard_query` function maps to the `wildcard_query` query used in search engine. It returns documents that match provided text in the specified field.
+OpenSearch supports wildcard characters `*` and `?`.  See the full description here: https://opensearch.org/docs/latest/opensearch/query-dsl/term/#wildcards.
+SQL wildcard character `%` is converted to `*`. SQL character wildcard `_` is converted to `?`. You may include a backslash `\` to escape wildcard characters.
```

Comment:
You need to escape characters in backticks to ensure that they are interpreted and rendered properly.
https://github.com/Bit-Quill/opensearch-project-sql/blob/1fb497379b5fb7a947e314d0862e59e39af31383/docs/user/dql/functions.rst#wildcard-query

Reply:
I'm not sure that we need to describe in user docs how it works under the hood.
I think you can simplify this line like this:
```
SQL wildcard characters `\%` and `\_` could be escaped by backslash `\\` if you need to perform exact search for those characters.
```
Please, correct wording and validate how escaping in RST works.

Reply:
Consider updating `LIKE` docs about this too.

Reply:
Thanks. Addressed.

Docs seem to cover like wildcards already https://github.com/opensearch-project/sql/blob/2.x/docs/user/dql/expressions.rst#like